PyNite/Mesh.py

```diff
@@ -1273,7 +1273,14 @@ def generate(self):
 
         radius = self.radius
         h = self.h
-        y = self.origin[1]
+
+        if self.axis == 'Y':
+            y = self.origin[1]
+        elif self.axis == 'X':
+            y = self.origin[0]
+        elif self.axis== 'Z':
+            y = self.origin[2]        
+
```

Comment:
I agree with these changes, but my copy of Pynite already has them. Someone else recently pointed this out and I fixed it. I need to update the PyPI version to include this change.